backend/charts/api/templates/deployment.yaml

```diff
@@ -80,6 +80,31 @@ spec:
             - migrate
             - up
 
+          {{- with .Values.migrations.extraEnvVars }}
```

Comment:
Hey so this isn't quite right as-is. 
The init containers doesn't have any env vars defined on it by default here so in order for this to generate valid yaml you'll have to add in the `env:` keyword that this will be nested under.

however, not sure if k8s likes having a potentially empty `env:` defined.
We could add an env var that is always there similar to theother container like this:

```
          env:
            - name: HOST_IP
              valueFrom:
                fieldRef:
                  fieldPath: status.hostIP
```
That way the `extraEnvVars` can just be conditionally added underneath of that, which makes the logic a lot simpler.